environment-dev.yml

```diff
@@ -7,6 +7,12 @@ dependencies:
   - cx_oracle=7.0
   - sqlalchemy=1.2.13
   - pyproj
+  - fiona
+  - pandas
+  - qgis=3.10
+  - statsmodels
+  - xlsxwriter
+  - xlrd
```

Comment:
Jeg synes du skal vente med dem her til du rent faktisk har tilføjet kode der skal bruge det. Gælder også `environment.yml`

Reply:
Med undtagelse af qgis naturligvis

Reply:
Jeg har (med undtagelse af Fiona, som jeg bruger i morgen) brugt alle de pakker der er tilføjet: Afløseren for fire-mark bruger dem, men jeg mente at det konceptuelt var at foretrække med et separat PR for korrektioner til opsætning og vejledning (bl.a. fordi jeg skal bruge en nogenlunde færdigbagt opsætningsinstruktion når jeg skal snakke med Tanya i morgen).

Ellers var min tanke bare at have et fuldt PR med to commits, hvor det ene er "instruktion og setup", det andet "dræb fire-mark", men retter til så de bliver helt adskilte.

Kan du ikke også lige kigge på Pythonversionen? .travis.yml siger 3.7, mens environment*.yml siger 3.6, og så vidt jeg kan se af releasenotes for 3.7 er der sket væsentlige forbedringer af typeannotering i 3.7. 

Så jeg har tænkt mig at rette til 3.7 Jeg har aldrig, under arbejdet med fire-mark, brugt andet end 3.7, så jeg er også lidt loren ved om min kode overhovedet holder til 3.6

(nyt commit på vej)

Reply:
Jeg tror ikke der er noget i vejen for at bruge Python 3.7. Vi valgte 3.6 i sin tid fordi vi ikke ville være helt cutting edge, men stadig gerne ville have f-strings. Nu er der gået en masse tid og 3.7 er ikke længere cutting edge, så det er vel meget passende at skifte til 3.7. Der kan potentielt komme nogle deprecation warnings men dem kan vi i så fald bare tage hånd om.

Det er rigtigt nok at Python 3.7 er system installationen på Travis men til at bygge dokumentationen bliver Python fra conda brugt. Så selvom det ser modstridende ud med de forskellige version så bruges altså bare den fra conda.